src/components/molecules/Reaction/Reaction.module.scss

```
@@ -0,0 +1,10 @@
+.wrapper {
+  display: flex;
+  justify-content: space-between;
+  align-content: center;
+}
+
```

```suggestion
.wrapper {
@include flexBetween 
}
```

mixin 으로 구성되어있는게 있어서 활용해도 좋을 것 같습니다!